File: domain/src/main/java/com/blackmirror/dongda/domain/model/WeChatLoginBean.java
package com.blackmirror.dongda.domain.model;

/**
 * Created by Ruge on 2018-04-19 下午6:09
 */
public class WeChatLoginBean extends BaseDataBean {

    public String auth_token;
    public String screen_name;
    public int has_auth_phone;
    public String current_device_type;
    public int is_service_provider;
    public String user_id;
    public String screen_photo;
    public String current_device_id;


    @Override
    public String toString() {
        return "PhoneLoginBean{" +
                "auth_token='" + auth_token + '\'' +
                ", screen_name='" + screen_name + '\'' +
                ", has_auth_phone=" + has_auth_phone +
                ", current_device_type='" + current_device_type + '\'' +
                ", is_service_provider=" + is_service_provider +
                ", user_id='" + user_id + '\'' +
                ", screen_photo='" + screen_photo + '\'' +
                ", current_device_id='" + current_device_id + '\'' +
                ", code=" + code +
                ", message='" + message + '\'' +
                ", isSuccess=" + isSuccess +
                '}';
    }
}

File: kdomain/src/main/java/com/blackmirror/dongda/kdomain/model/WeChatLoginBean.kt
package com.blackmirror.dongda.kdomain.model

/**
 * Created by Ruge on 2018-04-19 下午6:09
 */
class WeChatLoginBean : BaseDataBean() {

    var auth_token: String? = null
    var screen_name: String? = null
    var has_auth_phone: Int = 0
    var current_device_type: String? = null
    var is_service_provider: Int = 0
    var user_id: String? = null
    var screen_photo: String? = null
    var current_device_id: String? = null


    override fun toString(): String {
        return "PhoneLoginBean{" +
                "auth_token='" + auth_token + '\''.toString() +
                ", screen_name='" + screen_name + '\''.toString() +
                ", has_auth_phone=" + has_auth_phone +
                ", current_device_type='" + current_device_type + '\''.toString() +
                ", is_service_provider=" + is_service_provider +
                ", user_id='" + user_id + '\''.toString() +
                ", screen_photo='" + screen_photo + '\''.toString() +
                ", current_device_id='" + current_device_id + '\''.toString() +
                ", code=" + code +
                ", message='" + message + '\''.toString() +
                ", isSuccess=" + isSuccess +
                '}'.toString()
    }
}
